src/stylesheets/common/_alerts.scss

```diff
@@ -8,6 +8,14 @@
     text-decoration: underline;
     &:hover { color: inherit; }
   }
+  .btn {
+    text-decoration: inherit;
+    white-space: normal;
+  }
+  .btn-mapzen,
```

Comment:
Oh, right!  Good call.  I believe that was in the billing module on the dashboard.  I'm making some edits there, so I'll update here and there.